components/settings/billing/tab-navigation.tsx

```
@@ -1,199 +1,209 @@
-    </div>
-  );
+							return (
+								<button
+									key={tab.id}
+									onClick={() => onTabChange(tab.id)}
+									className={`flex items-center gap-3 px-4 py-3 rounded-lg transition-all duration-200 group ${
+										isActive
+											? 'bg-theme-primary-50 dark:bg-slate-800 text-theme-primary-700 dark:text-theme-primary-500 border-b-2 border-theme-primary-500'
+											: 'text-slate-600 dark:text-slate-400 hover:text-slate-900 dark:hover:text-slate-100 hover:bg-slate-50 dark:hover:bg-slate-700/50'
+									}`}
+								>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)